Python-21.py

```diff
@@ -0,0 +1,26 @@
+
+#leet code: house paint
+#time complexity: o(N)
+#Approach: First row remains the same and while calulating the second row  for red it takes the minimum of previous rows blue and green values + cost for painting red.
+
+
+def minCost(self, costs):
+        # write your code here
+        r=len(costs)
+        c=len(costs[0])
+        dp=[[0]*c for _ in range(r)]
+        dp[0][0]=costs[0][0]
+        dp[0][1]=costs[0][1]
+        dp[0][2]=costs[0][2]
+        
+        #edge case
+        if not costs:
+            return 0
+        
```

Comment:
try checking the egde cases in the start

Reply:
Changed the edge case. Thank you!